### ISSUE TITLE
Links are not highlighted 

### ISSUE THREAD

**Issue by christophM** [ropenscilabs/r-docker-tutorial#15]
All the links are with the same style as the normal text. This is coming from the dcTemplate: https://github.com/karthik/dcTemplate/blob/master/inst/rmarkdown/templates/dc_lesson_template/resources/style.css

Not sure if you want it like that, but I found it to be confusing. 

Also it would make sense to have the links open new tabs. To my knowledge that is not possible with Rmd. So the only solution for would be to directly write html code into the .Rmd files with target=_blank in the a tags.

**Comment by katieannemills** [comment]
I agree - there are a couple of issues with this lesson template (see #12); might consider our own fork since I know that template is more of a sketch / not actively maintained.

**Comment by bsmith89** [comment]
@marschmi and I are claiming this issue for the SWC Bug BBQ.
